### DISCUSSION TITLE
Unable to find basic.py

### DISCUSSION THREAD

**Discussion by D8ATech** [Textualize/textual#1018] [General]
I found in earlier discussions that the code for the application that you get when you run `python -m textual` is in a script called basic.py but I am unable to find this file anywhere in the repository.  GIthub searches show the discussions where this file is mentioned but shows that there is no file named basic.py

Is basic.py no longer available?  Has it been renamed to something else?

Thanks

**Comment by willmcgugan** [comment]
It’s demo.py

**Comment by D8ATech** [comment]
Found it thanks
